docs/installation/presto-connector-installation.rst

```diff
@@ -0,0 +1,36 @@
+
+==================
+Adding a Connector
+==================
+
+In Presto, connectors allow you to access different data sources -- e.g.,
+Hive, PostgreSQL, or MySQL.
+
+To add the Hive connector, create a file ``hive.properties`` in ``/etc/opt/prestoadmin/connectors``.
+
+For Cloudera CDH 5, it should have the following content: ::
+
+    connector.name=hive-cdh5
+    hive.metastore.uri=thrift://<metastore-host-or-ip>:<metastore-port>
+
+
+For Hadoop 2.0+ (including HDP), it should have the following content: ::
+
+   connector.name=hive-hadoop2
+   hive.metastore.uri=thrift://<metastore-host-or-ip>:<metastore-port>
+
+
+There are additional properties and possible configurations in the
+`Hive connector documentation <https://prestodb.io/docs/current/connector/hive.html>`_,
+for example if you have a HA Hadoop cluster.
```

Comment:
..for example, configurations that have an HA Hadoop cluster? 
Is "HA" pronounced "HA" or "H-A"? If "HA" it should be "a HA" and if "H-A" it should be "an HA".